modules/imaging_browser/php/mrinavigation.class.inc

```diff
@@ -27,6 +27,9 @@ namespace LORIS\imaging_browser;
 
 class MRINavigation
 {
+
+    var $urlParams = array();
```

Comment:
This is not the only class variable. Should we define all class variable here?

Reply:
Maybe, I was just adding the minimum changes needed here to include the Phan rule and fix the errors.